ga4gh/backend.py

```diff
@@ -218,6 +278,14 @@ def addDataset(self, dataset):
         self._datasetNameMap[dataset.getLocalId()] = dataset
         self._datasetIds.append(id_)
 
+    def addOntologies(self, ontologies):
```

Comment:
There can be more than one ontology per dataset.
I think this way is better and more generic. Will keep it like it is.